languages-and-frameworks/axum.html.markerb

```diff
@@ -0,0 +1,115 @@
+---
+title: "Run a Rust Axum App"
+layout: framework_docs
+sitemap: false
+order: 1
+---
+
+<% app_name = "hello-next" %>
+<%= partial "/docs/js/frameworks/partials/intro", locals: { runtime: "Rust Axum", link: "https://docs.rs/axum/latest/axum/" } %>
+
+Deploying an [Axum](https://docs.rs/axum/latest/axum/) app on fly.io is really straightforward! With the help of the wonderful [cargo chef](https://github.com/LukeMathWalker/cargo-chef) we get great build times and small images.
```

Comment:
```suggestion
Deploying an [Axum](https://docs.rs/axum/latest/axum/) app on Fly.io is really straightforward! With the help of the [cargo chef](https://github.com/LukeMathWalker/cargo-chef), we get great build times and small images.
```

Reply:
I'm suggesting we take out "wonderful", since these are docs; but we'll leave in the important bits about build times and small images!